src/cpp/libutil.cpp

```diff
@@ -41,8 +41,6 @@ ec_args             *el_create_ec_args(int k, int m, int w, int hd,
 
 ec_backend_id_t     get_ec_backend_id(int id) {
     switch(id) {
-        case 0:
```

Comment:
Not necessarily, you can always amend your PR/commits to reflect what is going on here.